openhands_cli/simple_main.py

```diff
@@ -142,6 +142,16 @@ def main() -> None:
             from openhands_cli.mcp.mcp_commands import handle_mcp_command
 
             handle_mcp_command(args)
+        elif args.command == "cloud":
```

Comment:
```suggestion
        elif args.command == "teleport":
```
How do you feel about “teleport”? 

“jump”
“transfer” (nope…)
“beam” (heh…)

Teleport is in CC, and it’s still new; and, it’s fun. You don’t forget it if you use it once, it will be _cool_… 🤔
Also, one could expect some users to ask; “does OH have teleport?”

Reply:
I'm good with this! cc @jpelletier1 for visibility and if you have any opinions

Reply:
Wait, I take it back… Teleport teleports the current conversation, it’s not when starting a new one…

Reply:
Or it could be both? Does it make sense to use both with the meaning “restore this conversation on the cloud” and “start this conversation on the cloud”?

Seems so to me 🤔

Reply:
I think we're only starting conversations on cloud for now, so I'll merge this as is

But in the future as the feature set expands, and if we restore conversations on cloud teleport might be more appropriate - wdyt?